content/actions/using-workflows/triggering-a-workflow.md

```diff
@@ -53,13 +53,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - env:
-          GITHUB_TOKEN: {% raw %}${{ secrets.MY_TOKEN }}{% endraw %}
+          GH_TOKEN: {% raw %}${{ secrets.MY_TOKEN }}{% endraw %}
           ISSUE_URL: {% raw %}${{ github.event.issue.html_url }}{% endraw %}
         run: |
           gh issue edit $ISSUE_URL --add-label "triage"
 ```
 
-Conversely, the following workflow uses `GITHUB_TOKEN` to add a label to an issue. It will not trigger any workflows that run when a label is added.
+Conversely, the following workflow uses `GH_TOKEN` to add a label to an issue. It will not trigger any workflows that run when a label is added.
```

Comment:
I think for this one, we still want `GITHUB_TOKEN` since it referring to the secret instead of the env variable name
```suggestion
Conversely, the following workflow uses `GITHUB_TOKEN ` to add a label to an issue. It will not trigger any workflows that run when a label is added.
```